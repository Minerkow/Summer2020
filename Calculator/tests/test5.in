a = ?;
b = ?;
c = ?;
d = ?;
e = a * b /c + d;
jv  = e -(405 *(430 /(393 -((((122 /245 )))-(816 *806 )*859 )/9 )*506 -451 )-211 *400 *(974 +339 )+581 /744 -616 );
afds  = 38 -(180 *(((891 +(556 )*274 )-290 /(979 *jv +jv /((538 *(168 )-jv ))))))/10000;
u  = 41 -jv *843 /(514 )/500;
jv  = e +36 *(816 /(909 +584 ))/jv ;
co  = 58 -((636 /575 /(((84 -jv /jv +jv /873 ))*jv +(552 ))));
print jv;
print afds;
print u;
print co;