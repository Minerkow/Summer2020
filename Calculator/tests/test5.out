4
-2720
3604
58
